prow/github/webhooks.go

```diff
@@ -38,44 +38,44 @@ func ValidateWebhook(w http.ResponseWriter, r *http.Request, hmacSecret []byte)
 
 	// Header checks: It must be a POST with an event type and a signature.
 	if r.Method != http.MethodPost {
-		responseHttpError(w, http.StatusMethodNotAllowed, "405 Method not allowed")
+		responseHTTPError(w, http.StatusMethodNotAllowed, "405 Method not allowed")
```

Comment:
Is this a `go lint` thing?  In the future let's do rote updates like that in a separate commit or, even better, a separate PR

Reply:
Yes it is. Ok!